src/control/lib/hostlist/hostlist_test.go

```diff
@@ -160,6 +160,17 @@ func TestHostList_Create(t *testing.T) {
 			expUniqOut:   "d-server-[0001-0002]ab",
 			expUniqCount: 2,
 		},
+		// TODO: fix irregularity
```

Comment:
Probably worth adding more detail to this comment unless you have an immediate followup. This is referring to the fact that the "negative port with ip" case doesn't return an error, right?

Reply:
I will elaborate